src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,20 +243,58 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
                     parts.AddLineBreak();
                     parts.AddText(WorkspacesResources.Exceptions_colon);
+                    var textSize = 0;
                     foreach (var exceptionString in exceptionTypes)
                     {
-                        parts.AddRange(LineBreak());
-                        parts.AddRange(Space(count: 2));
-                        parts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel));
+                        var checkedParts = new List<TaggedText>();
+                        checkedParts.AddRange(LineBreak().ToTaggedText());
+                        checkedParts.AddRange(Space(count: 2).ToTaggedText());
+                        checkedParts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel).ToTaggedText());
+                        var exceptionTexts = documentationComment.GetExceptionTexts(exceptionString);
+                        if (exceptionTexts.Length > 0)
+                        {
+                            var exceptionStr = $": {exceptionTexts[0]}";
+                            checkedParts.AddRange(formatter.Format(exceptionStr, symbol, _semanticModel, _position, ISymbolExtensions2.CrefFormat, CancellationToken));
+                        }
+
+                        for (var itemIndex = 0; itemIndex < checkedParts.Count; itemIndex++)
+                        {
+                            var item = checkedParts[itemIndex];
+                            if (textSize < maxTextSize)
+                            {
+                                if (textSize + item.Text.Length > maxTextSize)
+                                {
+                                    // truncate to 100.
+                                    // End the text with "..." and strip everything after 100 characters.
+                                    // should Remove's startindex just be "maxTextSize - 2"?
+                                    checkedParts[itemIndex] = item.WithText(item.Text.Remove((maxTextSize - textSize) - 2) + "...");
+
+                                    // update obtained item.
+                                    item = checkedParts[itemIndex];
+                                }
+
+                                textSize += item.Text.Length;
+                            }
+                            else if (textSize >= maxTextSize)
+                            {
+                                // Already at 100 so remove the rest.
+                                checkedParts.Remove(item);
+                            }
+                        }
+
+                        parts.AddRange(checkedParts);
                     }
 
-                    AddToGroup(SymbolDescriptionGroups.Exceptions, parts);
+                    _documentationMap.Add(SymbolDescriptionGroups.Exceptions, parts.ToImmutableArray());
```

Comment:
This change is surprising. What is it doing?

Reply:
I think it was because there was no overload of ``AddToGroup`` that would accept a list of ``TaggedText`` instances if I remember right back when the change was originally made (to convert it to an immutable array).

Reply:
Ah I remember, AddToGroup only takes in ``params SymbolDisplayParts[]`` and not ``TaggedText`` and still does.